gunicorn/arbiter.py

```diff
@@ -126,28 +125,23 @@ def start(self):
         """
         self.log.info("Starting gunicorn %s", __version__)
 
+        if 'GUNICORN_PID' in os.environ:
+            self.master_pid = int(os.environ.get('GUNICORN_PID'))
+            self.proc_name = self.proc_name + ".2"
+            self.master_name = "Master.2"
```

Comment:
It's not really a problem what we do here, but I wonder if maybe we just shouldn't rename the process. It's possible to tell which is which by the process tree and I'm just a little bit afraid the complexity isn't worth it. I can imagine bugs now or later where the process name ends up as "app.2.2.2.2.2". Maybe I'm just nervous because there was the recent tornado worker issue where the process name was incorrectly set.

I can see how it might make some shell script easier if you can grep for `.2$`, but if you're trying to automate this I think using the pid file is probably the better choice, anyway.